portfolio/src/main/webapp/contact.html

```diff
@@ -1,40 +1,42 @@
 <!DOCTYPE html>
-<meta charset="UTF-8">
-<title>Contact</title>
-<link rel="stylesheet" href="style.css">
-<script src="script.js"></script>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>Contact</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+<body>
+	<nav class="navbar">
+		<div class="nav-left-side">
+			<a href="/index.html" class="nav-button navbar-item">NR</a>
+			<a href="/works.html" class="nav-button navbar-item">Work</a>
+			<a href="/contact.html" class="nav-button navbar-item">Contact</a>
+		</div>
+		<div class="nav-right-side">
+			<a href="https://www.linkedin.com/in/roland-naijuka" target="_blank"
+				class="nav-button navbar-item"><img src="/images/linkedin.png" alt="My LinkedIn"></a>
+				<a href="mailto:naijuka.roland@outlook.com" target="_blank"
+					class="nav-button navbar-item"><img src="/images/mail.png" alt="Email me"></a>
+		</div>
+	</nav>
 
-<nav class="navbar">
-	<div class="nav-left-side">
-		<a href="/index.html" class="nav-button navbar-item">NR</a>
-		<a href="/works.html" class="nav-button navbar-item">Work</a>
-		<a href="/contact.html" class="nav-button navbar-item">Contact</a>
-	</div>
-	<div class="nav-right-side">
-		<a href="https://www.linkedin.com/in/roland-naijuka" target="_blank" class="nav-button navbar-item"><img src="/images/linkedin.png" alt="My LinkedIn"></a>
-		<a href="mailto:naijuka.roland@outlook.com" target="_blank" class="nav-button navbar-item"><img src="/images/mail.png" alt="Email me"></a>
+	<div class="contact-me">
+		<h1>Contact Me!</h1>
+		<form action="/data" method="POST">
+			<div class="form-content">
+				<label for="name">Name</label>
+				<input type="text" name="name" id="name" placeholder="optional"><br>
+                </div>
```

Comment:
the close tags (e.g. </div>) should be indented the same amount as the opening tag

Reply:
> the close tags (e.g. ) should be indented the same amount as the opening tag

Thank you for highlighting on this. I will make sure to keep track of this every time